contributed_definitions/nyaml/NXpiezoelectric_material.yaml

```diff
@@ -0,0 +1,93 @@
+category: base
+doc: |
+  Material description and properties of the piezoelectric scanner materials.
+  The material of the Piezoelectric scanner is composed of polycrystalline solids and 
+  attached at the head of the tip. The material shows mechanical deformation due to the applied electric field. 
+
+type: group
+NXpiezoelectric_material(NXobject):
+  # Online links:
+  #   1. ttps://www.ulprospector.com/knowledge/2689/pe-piezoelectric-materials/#:~:text=piezoelectric%20ceramics%2C%20such%20as%20lithium,as%20polyvinylidene%20fluoride%20(PVDF).
+  #   2. https://info.piezo.com/hubfs/Data-Sheets/piezo-material-properties-data-sheet-20201112.pdf
+  #   3. https://application.wiley-vch.de/books/sample/3527345124_c01.pdf
+
+  name(NX_CHAR):
+    doc: |
+      The name of the material of the piezo scanner such as Lead Zirconate Titanates (PZTs).
+  chemical_formula(NX_CHAR):
+    doc: |
+      The chemical formula of the material of the piezo scanner such as Pb(Zr,Ti)O3.#
+  type(NX_CHAR):
+    doc: |
+      The type of the material of the piezo scanner (e.g. piezoelectric ceramics, polymer-film piezoelectrics).
+  # Properties of the piezo material
+  density(NX_NUMBER):
+    unit: NX_DENSITY
+    doc: |
+      The density of the piezo material.
+    \@unit:
+      doc: |
+        The unit of the density of the piezo material.
+  relative_permittivity(NX_NUMBER):
+    doc: |
+      The relative permittivity of the piezo material.
+    \@unit:
+      doc: |
+        The unit of the relative permittivity of the piezo material.
+  D_piezoelectric_constant(NX_NUMBER):
+    doc: |
+      The piezoelectric charge coefficients of the materials. The coefficient is defined as the number of charges 
+      produced per unit area due to the force applied.
+    \@unit:
+      doc: |
+        The unit of the piezoelectric constant of the piezo material.
+  G_voltage_constant(NX_NUMBER):
+    doc: |
+      The field defines the electric field produced by per unit mechanical stress.
+    \@unit:
+      doc: |
+        The unit of the G voltage constant of the piezo material.
```

Comment:
The docs here (and in the following fields) are unneccessary, of course these are the units belonging to that particular concept.

Unit should be NX_VOLTAGE.

Reply:
Also, why are you writing `\@unit:` instead of `unit:`?